Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN buildDeps=' \
     \
     url="https://rakudo.org/dl/star/rakudo-star-${rakudo_version}.tar.gz" \
     keyserver='ha.pool.sks-keyservers.net' \
-    keyfp='ECF8B611205B447E091246AF959E3D6197190DD5 7A6C9EB8809CFEAF0ED4E09F18C438E6FF24326D B6F697742EFCAF5F23CE51D5031D65902E840821' \
+    keyfp='B6F697742EFCAF5F23CE51D5031D65902E840821' \
```

Comment:
Can you please justify this change?

Reply:
`B6F697742EFCAF5F23CE51D5031D65902E840821` is the only key that is valid even after November 29, 2020.

Reply:
Sorry to be insistent here. Can you please give me a reference for that?

Reply:
Execute `gpg --keyserver ha.pool.sks-keyservers.net --search-keys key_fingerprint`.

Reply:
Quite a bit of stuff when I run that. Let me parse it.

Reply:
I think the reason of the CI in docker-library/official-images#9037 is not successful is that it is importing the expired key.

Reply:
48 keys, actually. Anything that gives you direct access to the key?

Reply:
I think that the signature by Patrick Spek should still work, and you've removed that one, leaving the last one which seems to be a duplicate? Have you tried leaving just the one for @Tyil ? (BTW, I invoke you, we probably need your help here)

Reply:
```bash
$ gpg --keyserver ha.pool.sks-keyservers.net --search-keys ECF8B611205B447E091246AF959E3D6197190DD5
gpg: data source: http://209.244.105.201:11371
(1)     Steve Mynott <steve.mynott@gmail.com>
          256 bit EDDSA key 959E3D6197190DD5, created: 2018-01-22, expires: 2020-01-22 (expired)
Keys 1-1 of 1 for "ECF8B611205B447E091246AF959E3D6197190DD5".  Enter number(s), N)ext, or Q)uit >
$ gpg --keyserver ha.pool.sks-keyservers.net --search-keys 7A6C9EB8809CFEAF0ED4E09F18C438E6FF24326D
gpg: data source: http://209.244.105.201:11371
(1)     Naoum Hankache <naoum88@gmail.com>
          4096 bit RSA key 18C438E6FF24326D, created: 2018-11-29, expires: 2020-11-29
Keys 1-1 of 1 for "7A6C9EB8809CFEAF0ED4E09F18C438E6FF24326D".  Enter number(s), N)ext, or Q)uit >
$ gpg --keyserver ha.pool.sks-keyservers.net --search-keys B6F697742EFCAF5F23CE51D5031D65902E840821
gpg: data source: http://209.244.105.201:11371
(1)     Patrick Spek <p.spek@tyil.nl>
          4096 bit RSA key 7A6AC285E2D98827, created: 2018-09-04
Keys 1-1 of 1 for "B6F697742EFCAF5F23CE51D5031D65902E840821".  Enter number(s), N)ext, or Q)uit >
```

Reply:
OK, gotcha. Thanks a lot. Sorry for being so thick with this.